packages/es-dev-server/demo/syntax-error/syntax-error-resolve-imports.js

```diff
@@ -0,0 +1 @@
+import foo from './bar.js
```

Comment:
you forgot a quote here 
![image](https://user-images.githubusercontent.com/17054057/83238025-16542100-a196-11ea-9800-472ec8cae403.png)

Reply:
huhuhu